packages/data-stores/src/index.ts

```diff
@@ -8,6 +8,7 @@ import * as Launch from './launch';
 import * as Onboard from './onboard';
 import oneWeekPersistenceConfig from './one-week-persistence-config';
 import * as Plans from './plans';
+import * as WpcomPlansUI from './plans-ui';
```

Comment:
If we import it as `WpcomPlansUI`, which is a perfect name for me, the path should be `wpcom-plans-ui` to be consistent with the other modules.

Reply:
yeah, was thinking about what the final name would be. will update then and finalize. we can also update later if need to generalize beyond plans 🤔